cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/kind-cluster-setup.sh

```diff
@@ -5,88 +5,283 @@
 
 # this is used to build an image that can be used to stand-up a pod that serves static test-data in 
 # local kind cluster. Used by the integration tests. This script needs to be run once before the running
-# the test(s) 
+# the test(s). This script requires GitHub CLI (gh) to be installed locally. On MacOS you can
+# install it via 'brew install gh'
```

Comment:
I'd also add the link to the releases page: 
`https://github.com/cli/cli/releases`